vendor/modules.txt

```diff
@@ -8,7 +8,7 @@ github.com/Azure/go-ansiterm/winterm
 # github.com/MakeNowJust/heredoc v0.0.0-20170808103936-bb23615498cd
 ## explicit
 github.com/MakeNowJust/heredoc
-# github.com/Masterminds/goutils v1.1.0
+# github.com/Masterminds/goutils v1.1.1 => github.com/Masterminds/goutils v1.1.1
```

Comment:
This looks fun because before and after `=>` you have the same stuff, but if that works, that works :)